src/lib/OpenEXRCore/openexr_std_attr.h

```diff
@@ -50,7 +48,6 @@ worldToCamera
 worldToNDC
 deepImageState
 originalDataWindow
-dwaCompressionLevel
```

Comment:
was this deprecated earlier and is lingering?

Reply:
It's in a comment. Looks like @kdt3rd made a list of the standard optional attributes and left them in a comment.  Seems appropriate to remove them from there as well.